### ISSUE TITLE
error while running the container

### ISSUE THREAD

**Issue by amarts** [aravindavk/gluster-tester#4]
Found the below errors when I tried to run the command. (after fixing #3)

```
2019-05-30 11:36:32,618 - INFO - Started docker kill glusterfs-tester-1
2019-05-30 11:36:32,668 - WARNING - Failed docker kill glusterfs-tester-1. ignoring..
2019-05-30 11:36:32,668 - INFO - Output:
Error response from daemon: Cannot kill container: glusterfs-tester-1: No such container: glusterfs-tester-1

2019-05-30 11:36:32,668 - INFO - Started docker rm glusterfs-tester-1
2019-05-30 11:36:32,710 - WARNING - Failed docker rm glusterfs-tester-1. ignoring..
2019-05-30 11:36:32,710 - INFO - Output:
Error: No such container: glusterfs-tester-1

2019-05-30 11:36:32,710 - INFO - Started docker kill glusterfs-tester-2
2019-05-30 11:36:32,750 - WARNING - Failed docker kill glusterfs-tester-2. ignoring..
2019-05-30 11:36:32,750 - INFO - Output:
Error response from daemon: Cannot kill container: glusterfs-tester-2: No such container: glusterfs-tester-2

2019-05-30 11:36:32,751 - INFO - Started docker rm glusterfs-tester-2
2019-05-30 11:36:32,789 - WARNING - Failed docker rm glusterfs-tester-2. ignoring..
2019-05-30 11:36:32,789 - INFO - Output:
Error: No such container: glusterfs-tester-2

2019-05-30 11:36:32,790 - INFO - Started docker kill glusterfs-tester-3
2019-05-30 11:36:32,825 - WARNING - Failed docker kill glusterfs-tester-3. ignoring..
2019-05-30 11:36:32,825 - INFO - Output:
Error response from daemon: Cannot kill container: glusterfs-tester-3: No such container: glusterfs-tester-3

2019-05-30 11:36:32,825 - INFO - Started docker rm glusterfs-tester-3
2019-05-30 11:36:32,861 - WARNING - Failed docker rm glusterfs-tester-3. ignoring..
2019-05-30 11:36:32,861 - INFO - Output:
Error: No such container: glusterfs-tester-3

2019-05-30 11:36:32,861 - INFO - Started docker kill glusterfs-tester-4
2019-05-30 11:36:32,901 - WARNING - Failed docker kill glusterfs-tester-4. ignoring..
2019-05-30 11:36:32,901 - INFO - Output:
Error response from daemon: Cannot kill container: glusterfs-tester-4: No such container: glusterfs-tester-4

2019-05-30 11:36:32,901 - INFO - Started docker rm glusterfs-tester-4
2019-05-30 11:36:32,938 - WARNING - Failed docker rm glusterfs-tester-4. ignoring..
2019-05-30 11:36:32,939 - INFO - Output:
Error: No such container: glusterfs-tester-4

2019-05-30 11:36:32,939 - INFO - Started docker kill glusterfs-tester-5
2019-05-30 11:36:32,973 - WARNING - Failed docker kill glusterfs-tester-5. ignoring..
2019-05-30 11:36:32,974 - INFO - Output:
Error response from daemon: Cannot kill container: glusterfs-tester-5: No such container: glusterfs-tester-5

2019-05-30 11:36:32,974 - INFO - Started docker rm glusterfs-tester-5
2019-05-30 11:36:33,010 - WARNING - Failed docker rm glusterfs-tester-5. ignoring..
2019-05-30 11:36:33,010 - INFO - Output:
Error: No such container: glusterfs-tester-5

2019-05-30 11:36:33,010 - INFO - Started docker run -d --privileged=true --device /dev/fuse --name glusterfs-tester-1 --mount type=bind,source=/var/logs/gluster-tester/ld-1,target=/var/log --mount type=bind,source=/d/bd-1,target=/d gluster/glusterfs-tester:latest
2019-05-30 11:36:33,048 - ERROR - Failed docker run -d --privileged=true --device /dev/fuse --name glusterfs-tester-1 --mount type=bind,source=/var/logs/gluster-tester/ld-1,target=/var/log --mount type=bind,source=/d/bd-1,target=/d gluster/glusterfs-tester:latest
2019-05-30 11:36:33,049 - INFO - Output:
docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /var/logs/gluster-tester/ld-1.
See 'docker run --help'.
```

**Comment by amarts** [comment]
It went away after I created the directories `/d/bd-{1,2,3,4,5}` and `/var/logs/gluster-tester/ld-{1,2,3,4,5}`

**Comment by amarts** [comment]
Closed by #5
